packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/react';
+import React, { useMemo } from 'react';
+import ReactSelect, { createFilter } from 'react-select';
+
+import { SelectableValue } from '@grafana/data';
+
+import { withCenteredStory, withHorizontallyCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { Label } from '../Forms/Label';
+
+import { Select, VirtualizedSelect } from './Select';
+
+const meta: Meta = {
+  title: 'Forms/Select (Perf)',
+  decorators: [withCenteredStory, withHorizontallyCenteredStory],
+  argTypes: {
+    numberOfOptions: {
+      defaultValue: 10_000,
```

Comment:
is this some syntax trick i didn't know, or is just working by accident? 😱

Reply:
[Numeric separators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators)

Reply:
TIL